src/Services/Air/AirService.js

```diff
@@ -22,6 +22,28 @@ module.exports = function (settings) {
       debug,
       options
     ),
+    searchLowFaresAsync: uApiRequest(
+      config.AirService.url,
+      auth,
+      templates.lowFareSearch,
+      'air:LowFareSearchAsynchRsp',
+      AirValidator.AIR_LOW_FARE_SEARCH_REQUEST,
+      AirParser.AIR_ERRORS,
+      AirParser.AIR_LOW_FARE_SEARCH_REQUEST,
```

Comment:
What will return this parser in case of async request? Will it be possible to get `searchId` to use in `retrieveShop` function?

Reply:
Because the formatLowFaresSearch in AirFormat.js only returns list of fares, it is not possible to return the searchId without breaking the current structures, should I change the code to return optional like `return options.faresOnly ? response.fares : response;` ?